README.md

```diff
@@ -22,9 +22,68 @@ Once you join the The [Presto Foundation Slack Workspace](https://join.slack.com
+
+The current members of the Presto TSC are:
+
+| Name                      | Github                                              | Affiliation | Term begins  | Term ends    |
+| ------------------------- | --------------------------------------------------- | ------------|--------------|--------------|
+| Nezih Yigitbasi (chair)   |[@nezihyigitbasi](https://github.com/nezihyigitbasi) | Facebook    | 1 Jan, 2020  | TBD          |
+| Masha Basmanova           |[mbasmanova](https://github.com/mbasmanova)          | Facebook    | 1 Jan, 2020  | TBD          |
+| Zhenxiao Luo              |[zhenxiao](https://github.com/zhenxiao)              | Twitter     | 1 Jan, 2020  | TBD          |
+| Andrii Rosa               |[arhimondr](https://github.com/arhimondr)            | Facebook    | 1 Jan, 2020  | TBD          |
+| Rebecca Schlussel         |[rschlussel](https://.github.com/rschlussel)         | Facebook    | 1 Jan, 2020  | TBD          |
+| James Sun                 |[highker](https://github.com/highker)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Wenlei Xie                |[wenleix](https://github.com/wenleix)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Rongrong Zhong            |[rongrong](https://github.com/rongrong)              | Facebook    | 1 Jan, 2020  | TBD          |
+| Shixuan Fan               |[shixuan-fan](https://github.com/shixuan-fan)        | Facebook    | 1 Jan, 2020  | TBD          |
+| Jiexi Lin                 |[jessesleeping](https://github.com/jessesleeping)    | Facebook    | 1 Jan, 2020  | TBD          |
+
+Per the [Technical Charter](CHARTER.md), when there are less than six related companies represented on the TSC, the employees of each company may only submit one vote on TSC decisions where voting is required.
+
+## Policies and procedures
+
+The Presto TSC is governed by the [Presto Technical Charter](CHARTER.md).  The Charter provides foundational structure for the TSC on topics such as its scope, how to make decisions, and how to make changes to itself.  At the same time, it grants the TSC a high degree of freedom when determining how to implement the policies of the Presto Foundation.
+
+The following policies and procedures have been adopted by the TSC.
+
+### Term of TSC voting members
+
+The Presto TSC seeks to balance the energy and enthusiasm of new members with the stability provided by long-term members.  As such, TSC voting members will serve two-year terms, after which they will be eligible for re-election.  There is no limit to the number of times an individual may run for an open seat.
+
+To provide balance, the terms of TSC members will be adjusted so that roughly 50% of seats are elected each year.  If the classes become unbalanced, then the TSC shall designate the appropriate number of seats as one-year terms at the first opportunity to fix the imbalance.
```

Comment:
re-elect 1/3 of the TSC seats, so that TSC is stable?

Reply:
Either way is fine, although if all terms are the same length it would imply 3 year terms.  I might recommend starting with "how long should the a TSC serve before standing for reelection?" and working backwards from there.

Reply:
sure. nice